scripts/utilities.js

```diff
@@ -234,6 +234,55 @@ async function getBatchId(batchExchange) {
   return batchId.toNumber()
 }
 
+const WORD_DATA_LENGTH = 64
+const BALANCESTATES_STORAGE_SLOT = "0x0"
```

Comment:
ubernit: I would put this constant in `getBalanceState` since it is only applicable to reading from the user balances storage slot.